ephios/core/views/event.py

```diff
@@ -280,13 +280,19 @@ def _get_calendar_context(self):
             .prefetch_related("participations")
         )
         year, month = self.filter_form.get_calendar_month()
-        shifts = shifts.filter(start_time__year=year, start_time__month=month)
+        prevyear, prevmonth = _prevmonth(year, month)
+        nextyear, nextmonth = _nextmonth(year, month)
+
+        from_time = datetime.min.replace(year=year, month=month, tzinfo=get_current_timezone())
```

Comment:
pls add a comment explaining that we are not using the `__month` query because of mysql as we might stumble upon that in the future